### ISSUE TITLE
[Bug] Cannot read property stream of null

### ISSUE THREAD

**Issue by Mitchnsun** [shystruk/csv-file-validator#36]
### Current behavior:
I tried to test my code with cypress and [cypress-upload-file](https://github.com/abramenal/cypress-file-upload). But I have a parsing error (see below, cannot read stream of null) that I do not have outside of cypress.
During cypress test, I can console.log `file` (see code below) and cannot see a difference between in cypress or in a real browser...

I am not sure the issue is from csv-file-validator library, but I only have this issue during testing with cypress (I create an [issue](https://github.com/abramenal/cypress-file-upload/issues/183) on the repo of cypress-file-upload). Maybe the library waiting for something that the test do not provide.

### Desired behavior:
CSV-file-validator could parse the file.

### Steps to reproduce: (app code and test code)
Cypress: `cy.get('input[type=file]').attachFile('myfile.csv');`
Front: 
```
function onFileUploaded(event) {
    const file = event.target.files[0];
    CSVFileValidator(file, CSVConfig)
      .then(csvData => {
          /* Do stuff with csvData */
      })
      .catch(err => console.error(err) });
  };
}
<input type="file" accept=".csv" onChange={onFileUploaded} />
```

Trace error:
```
TypeError: Cannot read property 'stream' of null
    at Object.parse (papaparse.min.js:46)
    at csv-file-validator.js:17
    at new Promise (<anonymous>)
    at csv-file-validator.js:16
    at Cr (AccountantFileUpload.js:64)
    at Object.<anonymous> (react-dom.production.min.js:49)
    at d (react-dom.production.min.js:69)
    at react-dom.production.min.js:73
    at k (react-dom.production.min.js:140)
    at T (react-dom.production.min.js:169)
```

### Versions
Yarn: 1.12.3
Cypress: 4.3.0
Cypress-file-upload: 4.0.4
Csv-file-validator: 1.8.0
React: 16.6.0
MacOS: 10.14.6

**Comment by shystruk** [comment]
Hi @Mitchnsun 
Can you please check if you have a File in `onFileUploaded` method before call `CSVFileValidator`

**Comment by shystruk** [comment]
@Mitchnsun papaparse has been upgraded. You can get the latest version 1.8.0

**Comment by Mitchnsun** [comment]
Hi @shystruk, thanks for your answer.
I upgraded to 1.8.0 but unfortunately still have the issue.
Here is what I have before calling `CSVFileValidator` method :
<img width="1201" alt="Capture d’écran 2020-04-07 à 10 02 22" src="https://user-images.githubusercontent.com/1095047/78645649-23504480-78b8-11ea-891e-d092e5973789.png">
Pretty much the same in browser when it works smoothly.

**Comment by shystruk** [comment]
hm, the validator should work in this case. Waite for the cypress-file-upload response. 
Can you please create an example repo or send a demo code.

**Comment by Mitchnsun** [comment]
I may find something, in PapaParse they have this condition: `if ((global.File && _input instanceof File) || _input instanceof Object)` (l.246) `_input` is my `file` in this case.
*But* in cypress my `file` is not instanceof _File_, that's why it can parse it. I transfer this comment to cypress-file-upload.

**Comment by shystruk** [comment]
Good point. This may be the issue. Check on cypress-file-upload what instance of File they return and probably you can make it the File instance before you pass it to the validator.

**Comment by Mitchnsun** [comment]
They return an `object` not an `Object`. You can reproduce the bug with this repo: https://github.com/Mitchnsun/bug-cypress-csv-upload

I am not sure if the issue comes from cypress-file-upload or from Cypress and how they create a Blob/File.

I'll keep you updated

**Comment by shystruk** [comment]
As an idea try to get the file content from the object which cypress returns and create a File instance and pass it to the validator.

**Comment by shystruk** [comment]
closed the bug as it is not related to the validator

**Comment by Mitchnsun** [comment]
Thanks for your help, the issue was fixed by v4.0.5 of `cypress-upload-file`
